doc/api/vm.markdown

```diff
@@ -36,6 +36,12 @@ The options when creating a script are:
   code are controlled by the options to the script's methods.
 - `timeout`: a number of milliseconds to execute `code` before terminating
   execution. If execution is terminated, an [`Error`][] will be thrown.
+- `cachedData`: an optional `Buffer` with V8's code cache data for the supplied
+  source. When supplied `cachedDataRejected` value will be set to either
+  `true` or `false` depending on acceptance of the data by V8.
```

Comment:
@ofrobots This API could be used to produce such caching data. I guess we can just strip down the second commit and land it without it. Perhaps, V8 could be improved a bit, though, and rely only on the features that were actually used in the code, or allow compiling with reduced set of CPU features and using it on CPU that actually have wider feature support.